Mutex/functions/New-Mutex.ps1

```diff
@@ -105,7 +108,6 @@
             Name      = $newName
             Status    = "Open"
             Object    = $mutex
-            LockCount = 0
```

Comment:
Disagree with removing LockCount
Would be interested in the argument why you would want it removed